### ISSUE TITLE
Wooden post and breaker switch improvement.

### ISSUE THREAD

**Issue by tzpb8** [BluSunrize/ImmersiveEngineering#898]
Hi, i like to attach stuff on my wooden post LV Wire Connectors on the top and breaker switches on the sides below. But i cannot visually attach something on the sides of the wooden post, it will look like they are floating in the air because the post is thinner than a whole block size.

My suggestion is to somehow make it possible to attach things visually, directly on the post, no more floating Wire connectors, wall posts or breaker switches! It would be so cool to have it visually connect, maybe merge the block so it won't even take another block space?

The breaker gets integrated into the post? anything but floating in the air.

![2016-01-19_20 43 23 - kopia](https://cloud.githubusercontent.com/assets/9417797/12430081/ea6ea59a-beed-11e5-96ec-2cce7ffec8d4.png)

Oh and is there a way to use breaker switches without sending redstone signals? i just want to control the power but the redstone signal interrupts with some machines nearby, maybe a special breaker with no redstone output but a setting would probably be better. I see that there is settings already for inverting, maybe add another setting for no redstone signals? Also a way to rotate the breakers, if there is none already, i want to invert the vertical, but i cannot stand upside down to do that, i suggest the same mechanic as the wooden wall mount, place the breaker at the lower part of a block to have it face downwards, place it on the upper part to make it face upwards.

Thx :)

**Comment by BluSunrize** [comment]
Definitely not something I will work on for 1.7.10, major feature development has ceased.

**Comment by tzpb8** [comment]
Okay sounds good, it isn't super important or game breaking and its finally time to move on to 1.8.9 :D been waiting for so long for this.

Oh i guess i should mention that the wire connectors cannot send EU to the ic2 electric heat generator so i had to put a ic2 transformer on top of it. Maybe because they are the newest ic2 blocks or something, just unsupported, this had me a little puzzled at first and this heat mechanic seems cool.

I just made my first water mill too, sweet passive rf gain and it looks awesome!
I'll look into trippling it for more output.

A question tho: Does biodiesel count as biogas for the ic2 jetpack? i was thinking about building the biodiesel generator but wasn't sure if the jetpack would accept biodiesel as fuel. I finally solved this by spending a day fiddling around with ic2 stuff so i got my setup up and working now without biodiesel generator tho i wand to test all the machines in IE :D they seem to be cool, the bottler seems to have limited uses in infinity evolved mod pack tho, haven't built it yet.


More of my thoughts:

Thanks for this amazing mod, the wires if anything is a god sent and the capacitors too and i just discovered that the metal barrels have a bottom input/output slot! i was doing it with just the top slot before. so much better now! and everything just looks awesome :)
I need to start using conveyor belts more, i have one line setup for the crusher, so i don't have to jump around to get to the top and drop items manually that way lol.

Ahh that reminds me of the fluid pipes, when a pipe line is connected to multiple containers, they will leave 1mb of liquid behind at the original liquid storage for every extra connection.

So with 1 opentank and 3 cast basins, the smeltery would have 3mb remaining.
To fix, i have to disconnect all but one of the basins from the fluid pipe and it will drain the remainder when only 1 pipe remains. Annoying glitch here.

A way to get items out of containers horizontally would be great too, i don't always have the luxury of having an open bottom slot for a hopper.

Now that i think about it, the issue wasn't an IE machine, it was TE, so TE probably has some ways of dealing with this.
